packages/database/scripts/guard-local-migrate.mjs

```
@@ -0,0 +1,33 @@
+//
+// CockroachDB Cloud keeps an internal multi-region enum `crdb_internal_region`
+// even on single-region clusters. Prisma's `migrate dev` / `migrate status`
+// detect it as schema drift and try to `DROP TYPE "crdb_internal_region"`,
+// which fails with P3018 / SQLSTATE 2BP01. `migrate deploy` does not inspect
+// drift and is unaffected.
+//
+// Workflow: generate migrations with `prisma:migrate:diff` (no DB needed) and
+// apply them to the cloud with `prisma:deploy`.
+const url = process.env.DATABASE_URL ?? "";
+
+if (!/(localhost|127\.0\.0\.1)/.test(url)) {
```

ESLint

[error] Capturing group number 1 is defined but never used.

(regexp/no-unused-capturing-group)